packages/block-library/src/button/edit.js

```diff
@@ -130,6 +130,9 @@ class ButtonEdit extends Component {
 						<Dashicon icon="admin-links" />
 						<URLInput
 							value={ url }
+							/* eslint-disable jsx-a11y/no-autofocus */
```

Comment:
Hey! thanks for your first contribution. This change make sense to me. 

In general, when disabling an eslint rule, we add a comment explaining why we did so ("Disable reason").

Also, I think in this particular case, you're actually disabling the bad behavior (the usage of autoFocus). I think this probably should be the default in `URLInput` but this would be a breaking change that requires a bit more care so I think we can leave it for a potential separate PR.

Reply:
Thanks!
for this behavior, I tracked it back to this line
`autoFocus = _this$props$autoFocus === void 0 ? true : _this$props$autoFocus,`

setting the default value to false instead of true when `autoFocus` is not set, gives us the right behavior without using `autoFocus={ false }` on the button

Reply:
just to note, I can't find that line in the github logs or repository here but I can see it on the editor, I guess it has something to do with the build process
![image](https://user-images.githubusercontent.com/6165348/58752270-1b5b9300-84a4-11e9-9335-2f2c1b91ce3f.png)

Reply:
yes, you're looking at the built file and the source is a bit different. The fix is correct though, we just need to add a comment to explain why we're disabling the eslint rule.